### DISCUSSION TITLE
-new- Distinguishing Profile Feature

### DISCUSSION THREAD

**Discussion by StingX2** [RetroAchievements/RAWeb#808] [Ideas]
From 6/6/2021 Staff Meeting, Approved but agreed not possible until V2
(I'm sorry this isn't a greatly written proposal)

**8. Automated Ranking**
'5th in  Sets Made for PCEngine'
'25th Achievements Earned for PC Engine'
6th in Retro Ratio

the key is to not be spammy, whatevers decided to be a higher rank.

The key one I def want to do is have one for The top 25 points leaderboard, Top 100, and top 300. those are our big benchmarks on the global leaderboard

KickMeElmo — Today at 6:17 PM
Best way to do it would be to -be- spammy, but only show the top three or so without expanding the list.

DanielARP — Today at 6:18 PM
Seems like a good idea, would give quick insight on what type of player someone is :thumbsup:

**Comment by Sanaki** [comment]
As a point of clarification, this would be a list of rank-based accomplishments separate from badges, and would only be displayed if -currently- true. Losing the threshold rank would remove the line from the player's profile.
